recipes/moleculetda/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "moleculetda" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/moleculetda-{{ version }}.tar.gz
+  sha256: 43e2e2d21abbb1011421e98090013559c31658b350ce281ea874ec2a6c1dc1bb
+
+build:
+  entry_points:
+    - moleculetda = moleculetda.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win, py<36]
+
+requirements:
+  host:
+    - python 
+    - pip
+  run:
+    - python 
+    - scikit-learn
+    - dionysus
+    - numpy
+    - pyclustering
+    - pymatgen
+    - diode
+    - loguru
+    - click
+
+test:
+  imports:
+    - moleculetda
+  commands:
+    - pip check
+    - moleculetda --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/a1k12/molecule-tda
+  summary: Toolkit for using topological data analysis representations.
+  license: BSD-3-Clause
```

This isn't the standard BSD-3-Clause, but is:
```suggestion
  license: BSD-3-Clause-LBNL
```